src/System.Linq/tests/RangeRepeatTests.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public class RangeRepeatTests
+    {
+        #region Range
```

Comment:
My thoughts were: it is good to put these tests in a single file, because the test cases are very similar. But it is not a problem to split them into separate files. I'll do it.